.github/workflows/release.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
```

Comment:
Think we can only do `lit3-alpha` (as a temp situation.)
Once that becomes master, we remove `lit3` branch and create a `lit2` branch might we need it in the future (I suspect the need for it is way less then for our [`lit1`](https://github.com/ing-bank/lion/tree/lit1) branch)

Reply:
And once we create the lit2 branch, we should do this as well with a lit2 tag: https://github.com/ing-bank/lion/commit/8702381b52cd9ded130318397e3e3e5b30b17a4d

Reply:
Maybe it's also better to call the branch lit3-alpha instead of lit3 (for clarity)